src/sentry/integrations/slack/endpoints/event.py

```diff
@@ -127,19 +104,15 @@ def on_message(
 
         return self.respond()
 
-    def on_link_shared(
-        self,
-        request: Request,
-        slack_request: SlackRequest,
-    ) -> Optional[Response]:
-        matches: Dict[LinkType, List[UnfurlableUrl]] = defaultdict(list)
+    def on_link_shared(self, request: Request, slack_request: SlackDMRequest) -> bool:
+        """Returns true on success"""
```

Comment:
I think this is a good idea to use the `is_` prefix on boolean functions when the function has no side-effects. A more natural fit might be to raise exceptions where we would be returning `False`, but I also try to avoid using exceptions unless there is a genuine error. I kinda want to leave this as-is until we can refactor and overhaul these endpoints.